test_runner/fixtures/neon_fixtures.py

```diff
@@ -374,11 +421,14 @@ def __init__(
         run_id: uuid.UUID,
         mock_s3_server: MockS3Server,
         neon_binpath: Path,
+        compatibility_neon_binpath: Path,
         pg_distrib_dir: Path,
+        compatibility_pg_distrib_dir: Path,
         pg_version: PgVersion,
         test_name: str,
         top_output_dir: Path,
         test_output_dir: Path,
+        request: FixtureRequest,
```

Comment:
It doesn't seem that `request` belong to `NeonEnvBuilder` class.
`neon_env_builder` fixture should parse `combination` parameter and create `NeonEnvBuilder` with `version_combination` argument.

Reply:
fixed, thanks